smartapps/initialstate-events/initial-state-event-streamer.src/initial-state-event-streamer.groovy

```diff
@@ -389,13 +386,10 @@ def tryShipEvents() {
 			log.debug "shipped events and got ${resp.status}"
 			if (resp.status >= 400) {
 				log.error "shipping failed... ${resp.data}"
-			} else {
-				// clear the buffer
-				atomicState.eventBuffer = []
```

Comment:
No need to clear the buffer here on success since it will have already of been cleared before calling tryShipEvents to remove the chance of it possible growing unbounded if config is bad or if the eventBuffer grows too large to be sent to Initial State without error.